### PR TITLE
Implement login loader

### PR DESCRIPTION
**Pull request by alanhanaev**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -105,6 +105,8 @@ src/**/*.js
 !src/components/layouts/SecondMenu/SecondMenu.js
 !src/components/layouts/Header/Header.js
 !src/components/CompletedJobs/CompletedJobs.js
+!src/components/Login/LoginForm/LoginForn.js
```
Comment (sargonpiraev) [inline .eslintignore]: fix Forn

File: src/api/backend/index.js [hunk 2]
```diff
@@ -28,9 +28,9 @@ function deepSortByKey (obj) {
 export const signin = (account, roles) : SigninResBodyModel => {
   const body = { purpose: 'laborx' }
   if (roles) body.roles = roles
-  const data = JSON.stringify(deepSortByKey({ url: '/api/v1/security/signin/signature/laborx', body }))
```
Comment (sargonpiraev) [inline src/api/backend/index.js]: do not remove /laborx

File: src/api/backend/index.js [hunk 3]
```diff
@@ -28,9 +28,9 @@ function deepSortByKey (obj) {
...
+  const data = JSON.stringify(deepSortByKey({ url: '/api/v1/security/signin/signature', body }))
   const { signature } = account.sign(data)
-  return http.post('/security/signin/signature/laborx', body, {
```
Comment (sargonpiraev) [inline src/api/backend/index.js]: return /laborx

File: src/components/Login/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -47,17 +47,21 @@ class LoginForm extends React.Component {
...
+            type={Button.TYPES.SUBMIT}
+            label='Login'
+            primary
+            disabled={pristine || invalid}
+            error={error}
+            errorMods={css.errorForm}
+            mods={Button.MODS.INVERT}
+          />
+          :
+          <div className={css.spinner} />
```
Comment (sargonpiraev) [inline src/components/Login/LoginForm/LoginForm.js]: use material ui https://material-ui.com/demos/progress/ for spinner

File: src/components/Login/LoginForm/LoginForm.scss [hunk 5]
```diff
@@ -183,3 +183,27 @@
...
+@keyframes spinner {
+  0% {
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(360deg);
+    transform: rotate(360deg);
+  }
+}
```
Comment (sargonpiraev) [inline src/components/Login/LoginForm/LoginForm.scss]: rm 
use https://material-ui.com/demos/progress/ instead

File: src/components/Login/LoginForm/LoginForm.scss [hunk 6]
```diff
@@ -183,3 +183,27 @@
...
+  border: 0.125em solid #f3f3f3;
+  border-top: 0.125em solid #3498db;
+  border-radius: 50%;
+  width: 1em;
+  height: 1em;
+  animation: spinner 2s linear infinite;
+  display: inline-block;
+  margin: 0 .5rem;
+  margin-bottom: 2rem;
+}
```
Comment (sargonpiraev) [inline src/components/Login/LoginForm/LoginForm.scss]: del
use https://material-ui.com/demos/progress/ instead

**Review by sargonpiraev — CHANGES_REQUESTED** [review]

**Review by alanhanaev — COMMENTED** [review]
Requested changes

**Review by sargonpiraev — APPROVED** [review]

**Pull request merged**